src/platform/backends/libvirt/libvirt_virtual_machine.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2018 Canonical, Ltd.
+ * Copyright (C) 2019 Canonical, Ltd.
```

Comment:
Just curious why you didn't use `2018-2019` here since this file was created in 2018?

Reply:
Oops, I have also been doing this. I started after seeing it done in some change, I think...